src/files.js

```diff
@@ -2,6 +2,8 @@
 import { cozyFetch, cozyFetchJSON } from './fetch'
 import jsonapi from './jsonapi'
 import { DOCTYPE_FILES } from './doctypes'
+import qs from 'querystring'
```

Comment:
Does it work in a browser? Or does it mean that cozy-client-js will only be used in cozy-desktop and the konnectors?

Reply:
querystring should work in browsers : https://www.npmjs.com/package/querystring

Browser support is not optional, cozy-client-js is still used by frontend apps

Reply:
But, querystring must be in package.json for that to work, no?

Reply:
Yes it should, and it doesn't. Goog catch

Reply:
I think cozy-drive still uses it for some tasks. Then I will do it another way